src/styles/typography/index.md.njk

```diff
@@ -118,14 +118,23 @@ If you're displaying lots of links together and want to save space and avoid rep
   - adding a line of text before the links saying 'The following links open in a new tab'
   - including `<span class="govuk-visually-hidden">(opens in new tab)</span>` as part of the link text, so that part of the link text is visually hidden but still accessible to screen readers
 
+### Links on dark backgrounds
+
+Use the `govuk-link--inverse` modifier class to show white links on dark backgrounds - for example, in headers, custom components, and patterns with darker backgrounds.
+
+Make sure all users can see the links - the white links and background colour must have a contrast ratio of at least 4.5:1.
```

Comment:
We could link to the relevant WCAG success criteria here (https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) but maybe it doesn't add that much...

Reply:
```suggestion
Make sure all users can see the links - the white links and background colour [must have a contrast ratio of at least 4.5:1](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html).
```